### DISCUSSION TITLE
Why the gh copilot cli can not work when i installed it?

### DISCUSSION THREAD

**Discussion by IceRiverDev** [community/community#75715] [Copilot Conversations]
### github copilot cli

Question

I already installed the gh copilot cli extension, but when i executed the command like this: gh copilot suggest "Install git", there was no action happened on terminal, and then i  executed the echo $?, it return 255 code. Does somebody know why? BTW I already subscribed the github copilot before.

**Comment by iambashir** [comment]
Respons #1

**Comment by IceRiverDev** [comment]
After reinstalled the extension, the issue was gone, so this issue can be closed.
